public/js/pimcore/object/classes/data/multiselect.js

```diff
@@ -99,13 +103,147 @@ pimcore.object.classes.data.multiselect = Class.create(pimcore.object.classes.da
             return stylingItems;
         }
 
-        const valueStore = new Ext.data.JsonStore({
+        const optionsStore = new Ext.data.JsonStore({
             fields: ["key", {name: "value", allowBlank: false}],
             data: datax.options
         });
+        let optionsGrid = this.createoptionsgrid(optionsStore);
+        this.optionsModel = optionsGrid.getSelectionModel();
+
+        const defaultValueStore = new Ext.data.JsonStore({
+            fields: ["value"],
```

Comment:
wouldn't be better to be an extra column with a checkbox for every options (`field` in `optionsStore`) to avoid setting up default value that do not match any of the options, instead of own grid?